webapp/lib/app/users/view.dart

```diff
@@ -0,0 +1,259 @@
+import 'dart:html';
+import 'package:katikati_ui_lib/components/button/button.dart';
+import 'package:katikati_ui_lib/components/logger.dart';
+import 'package:katikati_ui_lib/datatypes/user.dart';
+import 'package:nook/view.dart';
+
+import 'controller.dart';
+
+Logger log = new Logger('view.dart');
+
+class Permission {
+  String key;
+  String type;
```

Comment:
can we make the `type` an enum containing `bool`, `int`, `string` and `set` ? We don't have uses for `int`s yet, but they can be helpful. We can also assume that `set`s will be `Set<String>` - if/when they're not, we'll work out what to do then)